### ISSUE TITLE
adding repositories from argued cli failed

### ISSUE THREAD

**Issue by dontreboot** [argoproj/argo-cd#2348]
**Describe the bug**

When I tried to add new repos from argocd cli, I was getting operation timed out error. My laptop didn't have direct access to the repo and it seemed that argocd cli was trying to access the repo directly from laptop; therefore, it timed out.
```
argocd repo add ssh://xxxx:7999/k8sc/calico.git --insecure-ignore-host-key --ssh-private-key-path ~/.ssh/id_rsa_argocd
FATA[0226] dial tcp 10.180.51.151:7999: connect: operation timed out
```

**To Reproduce**

1. add a repo using argued cli from a machine that doesn't have direct access to the repo
2. you are going to get timed out from argued

**Expected behavior**
argocd cli should add repo successfully as long as the argocd pod have access to the repo

**Version**

```
argocd: v1.0.2+e0bd546.dirty
  BuildDate: 2019-06-14T17:15:36Z
  GitCommit: e0bd546a07818ec06a27c2b3033454e3eb1c4152
  GitTreeState: dirty
  GoVersion: go1.11.4
  Compiler: gc
  Platform: darwin/amd64
argocd-server: v1.2.1+a6a394b
  BuildDate: 2019-09-12T17:16:02Z
  GitCommit: a6a394ba93a2a56981a45b9395e245e5210eaa35
  GitTreeState: clean
  GoVersion: go1.12.6
  Compiler: gc
  Platform: linux/amd64
  Ksonnet Version: 0.13.1
```

**Comment by alexec** [comment]
This maybe fixed in v1.3.

**Comment by dontreboot** [comment]
> This maybe fixed in v1.3.

When will v1.3 be released? Is this a known issue?

**Comment by dontreboot** [comment]
> It is fixed in ArgoCD 1.2 - please update your CLI to v1.2, too

cli 1.2.1 works. thanks!
```
argocd: v1.2.1+a6a394b
  BuildDate: 2019-09-12T17:16:32Z
  GitCommit: a6a394ba93a2a56981a45b9395e245e5210eaa35
  GitTreeState: clean
  GoVersion: go1.12.6
  Compiler: gc
  Platform: darwin/amd64
argocd-server: v1.2.1+a6a394b
  BuildDate: 2019-09-12T17:16:02Z
  GitCommit: a6a394ba93a2a56981a45b9395e245e5210eaa35
  GitTreeState: clean
  GoVersion: go1.12.6
  Compiler: gc
  Platform: linux/amd64
  Ksonnet Version: 0.13.1
```
